app/styles/ilios-common/components/sessions-grid-offering-table.scss

```diff
@@ -2,7 +2,7 @@
 
 .sessions-grid-offering-table {
   @include ilios-table-structure;
-  @include ilios-table-colors;
+  @include ilios-table-colors($blueMunsell);
```

Comment:
I'm hesitant to accept this because it appears to duplicate what is on line 11 of this file in setting the background color in the thead of this table? I think maybe the change should be to apply the color to [the th element](https://github.com/ilios/common/pull/2998/files#diff-8886b7715030234c793a0d7caaca81cb9d697ace4bef827b820416d0d0f83ef7L16) and not supply this as an option to the mixin or to remove the declaration from [the head](https://github.com/ilios/common/pull/2998/files#diff-8886b7715030234c793a0d7caaca81cb9d697ace4bef827b820416d0d0f83ef7L11) and use the mixin.

Reply:
i removed the redundant style declaration from row 11. that should fix this.

Reply:
:heavy_check_mark: